pkg/ebpf/events_definitions.go

```diff
@@ -22,16 +22,17 @@ type probe struct {
 	fn     string
 }
 
-type dependency struct {
-	eventID int32
+type dependencies struct {
+	eventIDs []int32
```

Comment:
Change to slice of struct named `eventsDependenceis` that include the event's ID